projects/ebv-analyzer-app/src/app/ebv-selector/ebv-selector.component.ts

```diff
@@ -209,6 +213,92 @@ export class EbvSelectorComponent implements OnInit, OnDestroy {
         return element.name.toLowerCase().includes(filterString);
     }
 
+    plot(): void {
+        combineLatest([
+            this.dataSelectionService.rasterLayer.pipe(
+                mergeMap<RasterLayer | undefined, Observable<RasterLayer>>((layer) => (layer ? of(layer) : of())),
```

Comment:
`of()` is the same as `EMPTY`, this never yields. Is that the intended purpose?

Reply:
Could you use a filter then instead? If not, we can keep it as is.

Reply:
I copied this code and I think it uses a map instead of a filter because it changes the type from `RasterLayer | undefined` to `RasterLayer` which I guess a filter does not.